GsaGH/Components/3_Loads/CreateGridPointLoad.cs

```diff
@@ -18,17 +20,37 @@
 
 namespace GsaGH.Components {
   public class CreateGridPointLoad : GH_OasysDropDownComponent {
+    private enum ExpansionType {
```

Comment:
duplicated in 3 classes

Reply:
updated: https://github.com/arup-group/GSA-Grasshopper/pull/445/commits/97910b3179f1b335ba8f848195d0735f94696bdb